data/documents-schema.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "$ref": "#/definitions/KlimawatchDocuments",
+  "definitions": {
+    "KlimawatchDocuments": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "entity": {
+                "$ref": "#/definitions/Entity"
+              },
+              "documents": {
+                "type": "array",
+                "items": {
+                  "$ref": "#/definitions/Document"
+                }
+              }
+            },
+            "additionalProperties": false,
+            "required": ["entity", "documents"]
+          }
+        }
+      },
+      "required": ["data"]
+    },
+    "Entity": {
+      "title": "Gebietskörperschaft: Landkreis, Gemeinde, etc",
+      "type": "object",
+      "properties": {
+        "id": {
+          "type": "string"
+        },
+        "name": {
+          "type": "string"
+        },
+        "adminLevel": {
+          "type": "integer"
+        },
+        "ags": {
+          "type": "string"
+        },
+        "wikidata": {
+          "type": "string",
+          "format": "uri"
+        },
+        "osm": {
```

Comment:
Ich verstehe, dass hier wahrscheinlich eine Verknüpfung mit einem Linienzug in OSM gemeint ist, der die Grenzen der Gebietskörperschaft beschreibt. Wollen wir das als `title` oder `description` aufnehmen?

Reply:
```suggestion
        "osm": {
          "description": "Querverlinkung auf OpenStreetMap Relation, dort kann z.B. die Geometrie abgerufen werden"
```